internal/p2p/peermanager.go

```diff
@@ -1419,47 +1419,6 @@ func (s *peerStore) Ranked() []*peerInfo {
 	}
```

Comment:
can we still skip over peers that are inactive? so we don't gossip them? Maybe  in a separate PR

Reply:
I think we do want to gossip inactive peers because if we can't dial them, someone else might for network routing reasons.

Reply:
I disagree. When we gossip the peers, the receiver doesn't know if they are invalid or not and will try to dial them. Of course, currently as it's just when they are on a different network, the node will find out immediately and mark them as inactive, I just think it's better if we don't propagate invalid addresses